pandas/core/frame.py

```diff
@@ -3384,7 +3384,7 @@ def nsmallest(self, n, columns, keep='first'):
         """
         return self._nsorted(columns, n, 'nsmallest', keep)
 
-    def swaplevel(self, i, j, axis=0):
+    def swaplevel(self, i=-2, j=-1, axis=0):
         """
         Swap levels i and j in a MultiIndex on a particular axis
```

Comment:
I think it's worth noting in the doc strings (of alll of these) that the defaults were added in 0.18.1

you can use a versionadded directive